### ISSUE TITLE
Feature-Request: Choosing the Image-Size of Exported Maps

### ISSUE THREAD

**Issue by gesundelehre** [felixhayashi/TW5-TiddlyMap#227]
Would it be possible to **add a functionality** that allows the user to **choose a size of the exported image**?

Reason: When trying to export large maps, the fontsize could be very small in the exported picture and thus the person cannot really see, what is written on it.

----
Suggestions:

  - Maybe it could be **directly added to** the **Dialog here**:
![](https://cloud.githubusercontent.com/assets/4307137/10394849/08a4de8e-6e9a-11e5-8154-3e059086d513.png)
  - Maybe a slider could be used or it could be chosen if the height or width can be entered by the user and the other property is being automatically calculated.

**Comment by felixhayashi** [comment]
Good point. I thought about this as well, the problem is that I did not find a way to resize the canvas programatically so far.

**Comment by ahilliard** [comment]
The progress on this has been very helpful - now being able to specify the canvas width and height is great.  Only undesired behavior I have seen is that if my canvas size has the wrong aspect ratio (e.g. too narrow), the exported image crops half the outermost nodes.  

Will look forward to testing this again once 0.17.8 hits http://tiddlymap.org/#Installation for us non Node.js users
